modules/metering-upgrade-verify.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-upgrading-metering.adoc
+
+[id="metering-upgrade-verify_{context}"]
+= Verify Metering Upgrade
+
+To verify your metering upgrade, check that all the required Pods have been created. Additionally, you can check that your `ReportDataSources` are importing new data.
```

Comment:
@lbarbeevargas I'm not sure `ReportDataSources` is a good check that upgrade was successful.

Reply:
@pruan-rht  I think this is still a good check to verify that we're still importing Prometheus metrics as intended.

Reply:
@timflannagan1 yeah, I just find that reading the timestamps kind of difficult since it's all in UTC.